FileShareService.DesktopClient.Core/Jobs/SetExpiryDateJob.cs

```diff
@@ -1,12 +1,21 @@
-﻿using System;
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
 
 namespace UKHO.FileShareService.DesktopClient.Core.Jobs
 {
     public class SetExpiryDateJob : IJob
     {
+        public const string JobAction = "setExpiryDate";
         public string DisplayName { get; set; }
 
         public SetExpiryDateJobParams ActionParams { get; set; } = new SetExpiryDateJobParams();
+        public List<string> ErrorMessages { get ; set ; }
```

Comment:
Please inline initialise `ErrorMessages` to avoid the compiler warning
````
public List<string> ErrorMessages { get ; private set ; } = new List<string>();

Reply:
Done